frontend/src/config/types.ts

```diff
@@ -442,6 +450,34 @@ export class WMSLayerProps extends CommonLayerProps {
   'thresholdValues'?: { label: string; value: string | number }[];
 }
 
+enum AggregationOptions {
+  PIXEL = 'pixel',
+}
+enum DateTypeOptions {
+  CONTINUOUS = 'continuous',
```

Comment:
@wadhwamatic what other date_type options do you foresee?

Reply:
So if startDate = endDate then DateTypeOption is single? Maybe adding DateTypeOption is not usefull?